.github/workflows/ci.yml

```diff
@@ -27,12 +27,17 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node_version: [12, 14, 16, 17]
+        node_version: [12, 14, 16, 17, 18]
         include:
           - os: macos-latest
             node_version: 16
+          - os: macos-latest
+            node_version: 18
```

Comment:
Do we want to test node18 on macos for all PR/commits? I thought we're doing cross OS tests for stable versions only.

Reply:
Node v18 will become the next LTS in half a year

Reply:
I see, sounds fair then to anticipate it 👍